app/views/publishers/vacancies/job_applications/_declined_row.html.slim

```diff
@@ -0,0 +1,3 @@
+- row.with_cell(text: govuk_link_to(application.name, organisation_job_job_application_path(vacancy.id, application)))
+- row.with_cell(text: application.email_address)
+- row.with_cell(text: application.declined_at&.to_fs(:day_month_year))
```

Comment:
if the row is 'declined' then declined_at must be set - see previous discussions about safe navigation operators